libs/experimental/langchain_experimental/autonomous_agents/autogpt/agent.py

```diff
@@ -133,11 +133,11 @@ def run(self, goals: List[str]) -> str:
                 f"Assistant Reply: {assistant_reply} " f"\nResult: {result} "
             )
             if self.feedback_tool is not None:
-                feedback = f"\n{self.feedback_tool.run('Input: ')}"
+                feedback = f"{self.feedback_tool.run('Input: ')}"
                 if feedback in {"q", "stop"}:
                     print("EXITING")  # noqa: T201
                     return "EXITING"
-                memory_to_add += feedback
+                memory_to_add += f"\n{feedback}"
```

Comment:
I added the newline down here to conform with existing behavior but could also remove that if it's not needed.